### ISSUE TITLE
Add Webmanifest / PWA Worker

### ISSUE THREAD

**Issue by kero3009destiny** [duolicious/duolicious-frontend#321]
There are FOSS turnkey solutions like PWABuilder, since;

  1. The app will be controversial for some time
  2. Some people do not trust even the established app-stores
  
 having a webapp may be good approach, user can get better experience without installing an actual app on their machine. Since PWAs can have workers the notifications can get a little more responsive, especially on iOS where they cannot exist without PWAs (I may be wrong.)

ref. 
https://web.dev/learn/pwa/progressive-web-apps
https://www.pwabuilder.com/

**Comment by duogenesis** [comment]
Good idea. Duolicious is built with Expo, which supports PWAs according to their [documentation](https://docs.expo.dev/guides/progressive-web-apps/). Their [notifications library](https://docs.expo.dev/versions/latest/sdk/notifications/) also seems to have added support for web browsers in the past few months, so hopefully it shouldn't be too difficult to implement this now.
